hack/unwanted-dependencies.json

```diff
@@ -180,21 +148,51 @@
       "go.opencensus.io": [
         "github.com/Microsoft/hcsshim"
       ],
+      "google.golang.org/appengine": [
+        "github.com/prometheus/client_golang"
+      ],
       "golang.org/x/exp": [
+        "github.com/Microsoft/hcsshim",
```

Comment:
can hcsshim stop using the unstable exp module?

Reply:
I don't think we directly are using this module anywhere. Its coming from google/x/sys and containerd/cgroups/v3

Reply:
@liggitt the previous discussion with cilium maintainers is here https://github.com/cilium/ebpf/issues/1095

Reply:
yeah, I know, I linked that below

looks like epbf is really close to not using the dep at all, I restarted the discussion at https://github.com/cilium/ebpf/issues/1095#issuecomment-2308009053 and opened https://github.com/cilium/ebpf/pull/1551 to propose removing the final use

Reply:
looks like this got cleaned up in https://github.com/cilium/ebpf/pull/1557

I wonder how close epbf is to a new tag... looks like we'd have to jump from v0.11.0 → v0.17.0 to pick that up... hopefully there's nothing else in that range that would be problematic to pick up